bindings/wallet-cordova/build_ios.py

```diff
@@ -52,4 +59,6 @@ def run():
 
 
 if __name__ == "__main__":
-    run()
+    # Set flag for emulator compatible lib
+    sim = "--sim" in sys.argv
```

Comment:
so we need to have plugin builds for simulators and builds for apps? I wonder if we can compile the two and choose one at (app) build time.

Reply:
Maybe I'll have to rename the target in order to have all into one lib. Will research that.

Reply:
All right, it may not be possible, though... In that case maybe we can put a hook on cordova or something